### PR TITLE
Made multisorting extractor only non-continuous

### PR DESCRIPTION
**Pull request by colehurwitz**
Removed continuous multisorting extractor (thank goodness lol)

### CODE REVIEW

**Comment by colehurwitz** [comment]
#193

**Comment by colehurwitz** [comment]
Any idea if this will effect any downstream functions? @alejoe91 I don't think anything was taking advantage of the continuous sorting extractor feature..

File: spikeextractors/MultiSortingExtractor.py [hunk 1]
```diff
@@ -51,107 +31,74 @@ def get_unit_spike_train(self, unit_id, start_frame=None, end_frame=None):
...
-    
+        if unit_id not in self._unit_map.keys():
+            raise ValueError("Non-valid unit_id")
+        sx = self._unit_map[unit_id]['sx']
+        unit_id_sx = self._unit_map[unit_id]['unit']
+        return self._SXs[sx].get_unit_property(unit_id_sx, property_name)
+
     def get_unit_property_names(self, unit_id=None):
-        if not self._allzeros:
+        if(unit_id is None):
```
Comment (alejoe91) [inline spikeextractors/MultiSortingExtractor.py]: PARENTHESIIIIIIISSSSSS!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

Comment (colehurwitz) [reply spikeextractors/MultiSortingExtractor.py]: XD

**Comment by colehurwitz** [comment]
I am gonna merge if this is alright.

**Pull request merged**